config/initializers/devise.rb

```diff
@@ -2,12 +2,21 @@
 
 Devise.setup do |config|
   # ==> Configuration for :database_authenticatable
-  # For bcrypt, this is the cost for hashing the password and defaults to 15. If
+  # For bcrypt, this is the cost for hashing the password and defaults to 11. If
```

Comment:
No, it was really 11. I increased it to 13 in the config. I don't know why I put the 15 there in the first place...

See https://github.com/plataformatec/devise/blob/f7b6d786066cef2f5e8d2ce9c6b6cc83918580eb/lib/generators/templates/devise.rb#L100